### ISSUE TITLE
Tray: Check on start if OpenPype is running already

### ISSUE THREAD

**Issue by m-u-r-p-h-y** [ynput/OpenPype#2666]
there is quite a common issue with nontechnical users, that they are trying to run OP and not noticing it has started successfully because of the hidden tray icon on windows.

I saw several cases with 30 instances of OP running at once.

It would be good to check if OP is running and notify the user it is running already on each attempt to start OP

 

[cuID:[OP-2600](https://app.clickup.com/t/256q8da)]

**Comment by BigRoy** [comment]
I tend to have multiple OpenPype instances open personally - even of just `tray`. A developer one, and a live production one. Would love to make sure I won't get many blocking messages like pop-ups or it not starting at all.

Maybe the better logic would be for `tray` to directly pop-up with something after start so that the user actually KNOWS it started aside of just the splash screen.

Would it just be a tray pop-up message bottom right saying something like:
> You currently have two running OpenPype Tray processes.

---

Does this problem only happen to Windows users due to "Show all icons in Taskbar" being disabled and them having not explicitly set OpenPype to show in taskbar by default?

**Comment by m-u-r-p-h-y** [comment]
Yes, it is caused by windows not showing all icons in the tray by default.

I can understand the need to run different versions at once. It is completely fine and we can just check if there is another OP of the same version running for example. Or is there any valid reason to run more OPs of the same version at once?

I do not want to make it more complicated, just want to help users to understand the situation better and prevent unexpected behavior with dozens of OPs running at once. . .
